platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java

```diff
@@ -180,6 +181,9 @@ private void onFailure(Exception e) {
         // PERMANENT_ERROR
         Timber.w("Request failed due to a permanent error: %s", errorMessage);
       }
+      if (logRequestUrl) {
+        Timber.v(call.request().url().toString());
```

Comment:
Can we give some more context to this log and maybe make it a `Timber.w()`? Alternatively, we could include it in above logs with simplified `if`.